Automattic/wp-calypso
# Implement marketplace yoast checkout thank you page

#### Changes proposed in this Pull Request
Add the post checkout thank-you page after the yoast purchase 
![image](https://user-images.githubusercontent.com/3422709/117968639-07816280-b344-11eb-9286-c12cf5dc5f2a.png)


#### Testing instructions

1. Create a site with a business plan and custom domain
2. Go to the domains page with the feature flag on. /plugins/domain/{siteUrl}?flags=marketplace-yoast
     * i.e. : http://calypso.localhost:3000/plugins/domain/awesome.com?flags=marketplace-yoast
     * i.e. : https://hash-462a273eeb0bd58aefdd9204e2216f189ca07f87.calypso.live/plugins/domain/awesome.com?flags=marketplace-yoast
3. Enter details and successfully complete checkout
4. The thank you page shown above should be shown
5. Verify all links point to the intended location (Get started, View posts, Ask a question, support documentation)

Related to #52081
